data/hai/hai outfits.txt

```diff
@@ -108,6 +134,65 @@ effect "ion impact"
 	"velocity scale" -.05
 
 
+outfit "Ionic Blaster Prototype"
+	category "Guns"
+	cost 530000
+	thumbnail "outfit/hai ionic blaster"
+	"mass" 19
+	"outfit space" -19
+	"weapon capacity" -19
+	"gun ports" -1
+	weapon
+		sprite "projectile/ion blast"
+			"frame rate" 5
+		sound "ion blast"
+		"hit effect" "ion impact"
+		"inaccuracy" 4
+		"velocity" 18
+		"lifetime" 30
+		"reload" 30
+		"firing energy" 72
+		"firing force" 12
+		"firing heat" 108
+		"hit force" 72
+		"shield damage" 70
+		"hull damage" 30
+		"ion damage" 1
+	description "The Unfettered conflict with the rest of the Hai has put pressure on them to gain a technological edge, despite their limited resources, to more effectively make their point. The Ionic Blaster is a result of that pressure. It is much smaller than the venerable Ion Cannon but still able to apply the same weapon jamming and energy draining effects to a useful degree. Some of that output has been traded to do enough raw damage to be useful in all situations."
+	description "	This aligns with the design goal for the Ionic Blaster, which was to produce a smaller ion weapon that can can fill all the gun ports on Shield Beetles with maximal efficiency, and still be useful on other smaller vessels. This goal has resulted in a design which cannot yet be mass produced, but is also small enough to be equipped on the Unfettered's new fighters. These weapons have consequently become the prized possessions of the few lucky enough to possess them."
+
+outfit "Ionic Turret Prototype"
+	category "Turrets"
+	cost 1260000
+	thumbnail "outfit/hai ionic turret"
```

Comment:
Because becca did not have the time to change that yet - these are not accessible in game though so it should be fine.
This PR was mostly done to figure out the stats of the weapons of the Unfettered campaign